content.properties

```diff
@@ -1,8 +1,26 @@
-#name=Ref 3.x distro
-#version=3.0.0
-#war.openmrs=2.6.1-SNAPSHOT
-omod.fhir2=1.9.0
-omod.webservices.rest=2.40.0-SNAPSHOT
-spa.frontendModules.@ohri/openmrs-esm-ohri-hiv-care-treatment-app=latest
-spa.frontendModules.@ohri/openmrs-esm-ohri-core-app=latest
-#spa.frontendModules.@openmrs/esm-primary-navigation-app=next
+# content.properties - Metadata for the content package and its dependencies
+
+# The name of this content package
+name=openmrs-content-hiv
+
+# The version of this content package (must follow SemVer)
+version=1.0.0-SNAPSHOT
+
+# Dependencies (specified in a format similar to distro.properties)
+
+# OMOD modules with their SemVer range
+omod.fhir2=1.0.0-SNAPSHOT
+omod.reporting=^2.0.0
+
+# SPA modules with their SemVer range
+spa.frontendModules.@openmrs/esm-login-app=^3.0.0
+spa.frontendModules.@openmrs/esm-patient-chart=^1.0.0
+
+# Additional dependencies can be listed here
+# omod.another-module=^X.X.X
+# spa.frontendModules.@openmrs/esm-another-app=^X.X.X
+
+# Ensure to follow SemVer ranges when specifying versions
+# For example:
+# - ^1.0.0 means compatible with 1.0.0 including patch updates
+# - ~1.2.3 means compatible with 1.2.x but not 1.3.0
```

Comment:
We can put this in docs rather than having it be part of this package.

Reply:
Will remove, left it there for demo purposes. Will put the file that's in the main branch